grpc.kafka.echo/README.md

```diff
@@ -0,0 +1,61 @@
+# grpc.kafka.echo
+
+Listens on https port `9090` and will exchange grpc message in probuf format through the `echo-commands` topic in Kafka.
+
+### Requirements
+
+- bash, jq, nc, grpcurl
+- Kubernetes (e.g. Docker Desktop with Kubernetes enabled)
+- kubectl
+- helm 3.0+
+
+### Setup
+
+The `setup.sh` script:
+- installs Zilla and Kafka to the Kubernetes cluster with helm and waits for the pods to start up
+- creates the `echo-commands` topic in Kafka.
```

Comment:
Given that the same topic is used for both request and response, suggest renaming to `echo-messages` instead of `echo-commands`.